### PR TITLE
Remove six from dependencies.

### PR DESCRIPTION
**Pull request by isavion**

### CODE REVIEW

**Comment by isavion** [comment]
anyone knows why CI isn't running for this PR?
I couldn't invoke the tests locally

File: anyway/flask_app.py [hunk 1]
```diff
@@ -131,8 +132,8 @@ def generate_csv(results):
             output.writeheader()
 
         row = {k: v.encode('utf8')
-               if type(v) is six.text_type else v
-               for k, v in iteritems(serialized)}
+        if type(v) is str else v
```
Comment (elazarg) [inline anyway/flask_app.py:135]: wrong indentation - this is part of the dict comprehension. Keep the indentation as it is.

File: anyway/flask_app.py [hunk 2]
```diff
@@ -153,13 +154,15 @@ def generate_csv(results):
...
+    kwargs = {arg: arg_type(request.values.get(arg, default_value)) for
+              (arg, (arg_type, default_value)) in
+              ARG_TYPES.items()}
     if request.values.get('age_groups[]') == '1234' or request.values.get('age_groups') == '1234':
         kwargs['age_groups'] = '1,2,3,4'
     try:
         kwargs.update(
-            {arg: datetime.date.fromtimestamp(int(request.values[arg])) for arg in ('start_date', 'end_date')})
+            {arg: datetime.date.fromtimestamp(int(request.values[arg])) for arg in
+             ('start_date', 'end_date')})
```
Comment (elazarg) [inline anyway/flask_app.py:165]: This is unrelated to six, I assume. I think it's best not to add unnecessary diffs; if at all, cosmetic changes should be added after the approval.

File: anyway/flask_app.py [hunk 3]
```diff
@@ -297,7 +300,8 @@ def news_flash():
         news_flash_obj = news_flash_obj.filter(
             NewsFlash.id == news_flash_id).first()
         if news_flash_obj is not None:
-            return Response(json.dumps(news_flash_obj.serialize(), default=str), mimetype="application/json")
+            return Response(json.dumps(news_flash_obj.serialize(), default=str),
+                            mimetype="application/json")
```
Comment (elazarg) [inline anyway/flask_app.py:304]: same

**Comment by elazarg** [comment]
LGTM in general, except the minor changes (indentation etc.) which are unrelated to the core changes and might hinder future reviews (e.g. git blame). I'd suggest separating those to an independent commit, to be pushed after the approval.

**Comment by atalyaalon** [comment]
@elazarg Thank you for the great review!
@isavion please perform the changes as Elazar requested

**Comment by atalyaalon** [comment]
And of course - @idosavion great work!

**Comment by elazarg** [comment]
`enum34` can probably also be removed.

**Comment by idosavion** [comment]
> LGTM in general, except the minor changes (indentation etc.) which are unrelated to the core changes and might hinder future reviews (e.g. git blame). I'd suggest separating those to an independent commit, to be pushed after the approval.
@elazarg Great point!
Fixed required changes.

**Comment by atalyaalon** [comment]
@idosavion tests are not passing - please recheck pr

**Comment by idosavion** [comment]
fixed

File: anyway/flask_app.py [hunk 4]
```diff
@@ -131,8 +130,8 @@ def generate_csv(results):
             output.writeheader()
 
         row = {k: v.encode('utf8')
-               if type(v) is six.text_type else v
-               for k, v in iteritems(serialized)}
+               if type(v) is str else v
```
Comment (elazarg) [inline anyway/flask_app.py:133]: That's weird (the original test, too). 
```suggestion
               if isinstance(v, str) else v
```

Comment (idosavion) [reply anyway/flask_app.py:133]: I have a few other fixes I want to add to this file, but as you suggested, I'll add them in a separate PR.

**Comment by elazarg** [comment]
Some tests still use `six`. For example [test_flask.py](https://github.com/hasadna/anyway/blob/fcb1da32a69b9d59fbf44ed404545b7564ae51a6/tests/test_flask.py#L8)

**Comment by atalyaalon** [comment]
@isavion please create a new PR replacing the rest of the six dependencies in:
anyway/parsers/cbs.py
anyway/flask_app.py
